### ISSUE TITLE
doc mentions obsolete relative_number feature

### ISSUE THREAD

**Issue by shuttie** [metarank/metarank#1187]
The scaling is natively done by xgboost/lightgbm, so no need to explicitly normalize numerical data - it's done automatically for all numbers.
